packages/app/documents/__tests__/document.test.ts

```diff
@@ -76,6 +76,10 @@ describe('Documents', () => {
             'publishMessageToQueueChannel'
         )
 
+        queueSpy.mockImplementation(async () => {
```

Comment:
For this suite of tests, we had no mock implementation for `publishMessageToQueueChannel`, which was causing the tests to time out while awaiting the result of that publication. All unit tests now pass.